packages/admin/src/styles/admin-m3-components.css

```
@@ -0,0 +1,230 @@
+ * only where a variant is centered (compact mobile dialogs and all desktop
+ * dialogs); full-width mobile action sheets have no X translate. Tailwind v4
+ * compiles translate utilities to the independent `translate` CSS property —
+ * NOT `transform`. Because `translate` and `transform` compose, these keyframes
+ * must NOT re-bake translate(-50%, …) into `transform`, or centered variants
+ * would translate twice and land off-center.
+ * ============================================================================ */
+
+/* Mobile bottom sheet — slide up from below, 2% overshoot for spring feel.
+ * X placement comes from AdminDialog.tsx geometry; we only move Y. */
+@keyframes adminDialogSheetIn {
+  from { transform: translateY(100%); opacity: 0.85; }
```

Stylelint (17.14.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)